src/containers/snapshot/snapshot.jsx

```diff
@@ -146,15 +142,13 @@ class Snapshot extends Component {
                         <Grid item xs={5} sm={4} md={3} lg={2}>
                             <ItemCollection 
                                 items={this.props.real.items} 
-                                unsnapshot_items={unassigned_items} 
-                                getDragItemColor={this.getDragItemColor} />   
+                                unsnapshot_items={unassigned_items} />   
                         </Grid>
                         <Grid item xs={7} sm={8} md={9} lg={10}>
                             <ContainerCollection 
                                 snapshot={this.getSnapshot(this.props.snapshotId)} 
                                 containers={this.props.real.containers} 
-                                items={this.props.real.items} 
-                                getDragItemColor={this.getDragItemColor} />
+                                items={this.props.real.items} />
```

Comment:
what is this dragItemColor? how come we don't need it anymore?

Reply:
Because we don't need the item being dragged to have a color.